test/unit/css-test.js

```diff
@@ -26,17 +26,17 @@ define([
 
         // check that style sheet is correctly included to the end of head
         var styleSheet = document.getElementsByTagName('head')[0].lastChild;
-        assert.ok(styleSheet.innerHTML.indexOf('test-selector{background-color:blue}') >=0 , 'css correctly included');
+        assert.ok(styleSheet.innerHTML.indexOf('test-selector{background-color: blue;}') >=0 , 'css correctly included');
```

Comment:
Let's use RegEx for these.

And make 2 tests:

1. `utils.css(el, { styles object })`
1. `utils.css(el, " styles string ")`